packages/calypso-e2e/src/lib/blocks/block-flows/contact-info.ts

```diff
@@ -51,12 +51,12 @@ export class ContactInfoBlockFlow implements BlockFlow {
 		const numericOnlyPhoneNumber = this.configurationData.phoneNumber.replace( /\D/g, '' );
 
 		const expectedEmailLinkLocator = context.page.locator(
-			`[href="mailto:${ this.configurationData.email }"]`
+			`main [href="mailto:${ this.configurationData.email }"]`
```

Comment:
**Suggestion (non-blocking)**: In all these cases where we have to specify `main`, I would recommend adding a comment about the sneaky debug elements! I could see someone wondering why we have to specify the `main` tag, and trying to remove it.

Reply:
Makes sense! I have added a comment in `whatsapp-button.ts` in this commit [e6bc28a](https://github.com/Automattic/wp-calypso/pull/69509/commits/e6bc28aac9dc01087e4df356d68f678393b5b6a5).